src/protected/application/lib/modules/Diligence/views/tado/emitir.php

```diff
@@ -24,7 +23,7 @@
             <header class="main-content-header">
                 <div class="header-content">
                     <div class="avatar">
-                        <img class="js-avatar-img" src="http://0.0.0.0:8088/assets/img/avatar--project-636617000-1709737773.png">
+                        <img class="js-avatar-img" src="<?= $app->getAssetUrl(); ?>/img/avatar--project-636617000-1709737773.png">
```

Comment:
Acredito que esse nome de arquivo se altera cada vez que ocorre um build novo na aplicação.
Recomendo a seguinte alteração:
```suggestion
                        <img class="js-avatar-img" src="<?php echo $this->asset('img/avatar-project.png', false); ?>">
```